macosx/mac-render.mm

```diff
@@ -297,6 +297,10 @@ void S9xPutImage (int width, int height)
         for (int i = 0; i < 60; i++)
             frameCalc += drawnFrames[i];
 
+		// avoid dividing by 0
+		if (frameCalc == 0)
+			frameCalc = 1;
+		
         IPPU.DisplayedRenderedFrameCount = (Memory.ROMFramesPerSecond * 60) / frameCalc;
```

Comment:
When macFrameSkip is set to 0, frameCalc will try to divide by 0 and throw an error here. I've set this as a placeholder for now, but I'm assuming there's a better way to do this?

Reply:
Yeah, that's an odd one.

I think the answer here is in S9xSyncSpeed(), instead of always setting `skipFrames` to `macFrameSkip`, we should set it to 1 when `macFrameSkip` is 0.

This code predates me, but it appears that `skipFrames` is only used to determine the emulated (but not necessarily drawn) frames per second. It should always be set to 1 when the frame was drawn without skipping, and set to the number of skipped frames + 1 otherwise.